### ISSUE TITLE
Voice volume depends on MP3Volume

### ISSUE THREAD

**Issue by a1batross** [FWGS/xash3d-fwgs#1744]
Because we put voice data to stream buffer, it's dependent on MP3Volume. 

It's possible to direct it to room buffer, but then it will be affected by DSP effects.

It's also possible to write it to the resulting buffer, but it doesn't obey master volume.

**Comment by a1batross** [comment]
Not anymore. Voice chat volume is now only controlled by `voice_scale` cvar, thanks to @pwd491

**Comment by rootcreep** [comment]
> Not anymore. Voice chat volume is now only controlled by `voice_scale` cvar, thanks to [@pwd491](https://github.com/pwd491)

I would open a new issue, as it seems to me that it is worth working on. A separate voice buffer is not what the original GoldSrc does. At least, I'm increasingly encountering the problem of excessive voice volume from some players who use a high-quality microphone and may be programmatically amplifying the microphone signal. The goldsrc engine skillfully mixes the sounds of the voice with the master volume, and there are no such problems. But so far, yes, it seems to be working.

**Comment by a1batross** [comment]
@pwd491 you're right, voice impl seems to be much more complicated in GoldSrc/Source. I'm also not sure if we should use `voice_scale` for that purpose. Maybe when mixing up voicebuffer we should take master volume into account (thankfully, it can be done with one-liner).

>At least, I'm increasingly encountering the problem of excessive voice volume from some players who use a high-quality microphone and may be programmatically amplifying the microphone signal

Yeah, GoldSrc/Source have the balancer that dynamically amplifies or reduces the input signal and we had that implemented but it only resulted in worse quality overall.
